panamaconverter/package.mill

```diff
@@ -17,12 +18,13 @@ trait PanamaConverter
     with HasPanamaOMModule
     with CrossModuleBase
     with HasScala2Plugin
-    with ScalafmtModule {
+    with ScalafmtModule
+    with release.ChiselPublishModule {
   def millSourcePath = super.millSourcePath / os.up
 
   def panamaOMModule = panamaom.cross(crossScalaVersion)
   def chiselModule = chisel(crossScalaVersion)
   def pluginModule = plugin.cross(crossScalaVersion)
 
-  override def moduleDeps = super.moduleDeps ++ Some(chiselModule)
+  override def compileModuleDeps = super.compileModuleDeps ++ Some(chiselModule)
```

Comment:
Test of panama should explicitly add `chiselModule` to `moduleDeps`

Reply:
Propagating dependencies manually requires the user to add chisel as a dependency to all modules that depend on panama, which is cumbersome.
Maybe we can inject the classpath to the module at compile time and use unipublish to propagate the Ivy dependency downstream.